packages/frontend/src/features/sqlRunner/components/SaveSqlChartModal.tsx

```diff
@@ -76,17 +75,25 @@ export const SaveSqlChartModal: FC<Props> = ({ opened, onClose }) => {
         if (spaces.length === 0) {
             return;
         }
+        let newSpace = form.values.newSpaceName
```

Comment:
this is great, but we should set the `initial space` on the `useEffect` where we set the initial name & description.

Something that we have on `SaveToSpaceOrDashboard`

```tsx

 let initialSpaceUuid;

            const isValidDefaultSpaceUuid = spaces.some(
                (space) => space.uuid === defaultSpaceUuid,
            );

            if (spaces && spaces.length > 0) {
                initialSpaceUuid = isValidDefaultSpaceUuid
                    ? defaultSpaceUuid
                    : spaces[0].uuid;
            }
```

could you add something like this?

Reply:
updated